sklearn/utils/random.py

```diff
@@ -23,9 +23,10 @@ def _random_choice_csc(n_samples, classes, class_probability=None,
     classes : list of size n_outputs of arrays of size (n_classes,)
         List of classes for each column.
 
-    class_probability : list of size n_outputs of arrays of size (n_classes,)
-        Optional (default=None). Class distribution of each column. If None the
-        uniform distribution is assumed.
+    class_probability : list of size n_outputs of arrays of size (n_classes,),\
+        default=None. 
+        Class distribution of each column. If None uniform distribution is 
+        assumed.
```

Comment:
```suggestion
    class_probability : list of size n_outputs of arrays of \
        shape (n_classes,), default=None. 
        Class distribution of each column. If None uniform distribution is 
        assumed.
```

Reply:
You can also use "shape" instead of "size" for the array in `classes`, as we do in other places.

Reply:
Ah, right, thanks @adrinjalali, committing the change.